assets/scripts/lobby/options.js

```diff
@@ -792,4 +819,22 @@ function updateCompactView(input){
 
 
 
-}
\ No newline at end of file
+}
+
+function updateGunImage(input) {
+    if (input === "shield") {
+        //when shields are used
+        $(".gunImg").attr('src','pictures/shield.png');
+        docCookies.setItem('gunPos.x', 5);
+        docCookies.setItem('gunPos.y', 0.8);
+        docCookies.setItem('optionProposedTeamIcon', 'shield');
```

Comment:
Is there a way to not store gunPos variables in cookie? Also please set the optionDisplayProposedTeamIcon cookie above in the initialiseEventListener area :)

Reply:
I will try to store it inside optionDisplayProposedTeamIcon cookie instead.

Accepted. Will be resolved in the next commit